### ISSUE TITLE
Renderer breaks build on aarch64-linux

### ISSUE THREAD

**Issue by donovanglover** [danyspin97/wpaperd#104]
Thank you for making wpaperd! Seems like [new rendering code](https://github.com/danyspin97/wpaperd/blame/b1d418fb6f44dce7df8563ff4d75f598c4ff4a26/daemon/src/render/renderer.rs#L436-L442) is causing the build to fail on aarch64-linux.

```rust
error[E0308]: mismatched types
   --> daemon/src/render/renderer.rs:436:68
    |
436 |         let vertex_shader = create_shader(gl, gl::VERTEX_SHADER, &[VERTEX_SHADER_SOURCE.as_ptr()])
    |                                                                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected `*const i8`, found `*const u8`
    |
    = note: expected raw pointer `*const i8`
               found raw pointer `*const u8`
error[E0308]: mismatched types
   --> daemon/src/render/renderer.rs:442:15
    |
442 |             &[FRAGMENT_SHADER_SOURCE.as_ptr(), shader.as_ptr()],
    |               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected `*const i8`, found `*const u8`
    |
    = note: expected raw pointer `*const i8`
               found raw pointer `*const u8`
For more information about this error, try `rustc --explain E0308`.
error: could not compile `wpaperd` (bin "wpaperd") due to 2 previous errors
```

**Comment by danyspin97** [comment]
Hello @donovanglover and thank you for the issue! This bug has just been fixed in the main branch and will be included in release 1.1.1 shortly :)

**Comment by danyspin97** [comment]
Closed as 1.1.1 has been released with the fix.
